Doc/c-api/call.rst

```diff
@@ -93,7 +93,7 @@ This is a pointer to a function with the following signature:
    and they must be unique.
    If there are no keyword arguments, then *kwnames* can instead be *NULL*.
 
-.. c:macro:: PY_VECTORCALL_ARGUMENTS_OFFSET
+.. data:: PY_VECTORCALL_ARGUMENTS_OFFSET
```

Comment:
```suggestion
.. :c:var:: PY_VECTORCALL_ARGUMENTS_OFFSET
```

Assuming this is a constant flag, shouldn't this be C-domain, not Python-domain? (There's no `:data:`, but there's `:var:` which the semantic equivalent for C in Sphinx.)

Reply:
Silly Sphinx-quesion: Why is `PY_VECTORCALL_ARGUMENTS_OFFSET` `:data:` here and `:const:` in the NEWS entry?

Reply:
If you look at the definition of other similar macro constants, you can see that they are defined as `:data:` and then referenced as `:const:`. I am trying to copy that pattern onto `PY_VECTORCALL_ARGUMENTS_OFFSET`. Currently, that is not the case, and you can see on the Python website that cross-references involving this specific macro constant are broken while others work.